### DISCUSSION TITLE
Upgrade from default scheduler to topolvm-scheduler?

### DISCUSSION THREAD

**Discussion by guettli** [topolvm/topolvm#919] [Q&A]
We plan to start with the default scheduler (Storage Capacity Tracking) and plan to use the topolvm-scheduler later.

Is this upgrade possible on a production cluster?

**Comment by cupnes** [comment]
I will provide a response by next week.

**Comment by cupnes** [comment]
@guettli 
I think it's possible. I confirmed that when the default scheduler (Storage Capacity Tracking) is set up, topolvm can be configured with Helm and the topolvm-scheduler can be enabled. (In that state, the e2e tests passed.) However, it is recommended to sufficiently test it before trying it in the production cluster.

**Reply by guettli** [reply]
thank you, for your answer!
